### PR TITLE
chore: Update golang.org/x/* modules

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

Update modules golang.org/x/* to the latest versions.

**Release note**:

```
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -79,12 +79,12 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/multierr v1.7.0 // indirect
 	go.uber.org/zap v1.19.1 // indirect
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
+	golang.org/x/net v0.17.0 // indirect
 	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b // indirect
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
```
Comment (0xFelix) [inline go.mod:84]: Can you upgrade these too please?

Comment (opokornyy) [reply go.mod]: Done

**Comment by ksimon1** [comment]
@opokornyy can you please fix this error:
```
Error: err: exit status 1: stderr: go: inconsistent vendoring in /go/src/github.com/kubevirt/tekton-tasks-operator/api:
	golang.org/x/net@v0.17.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	golang.org/x/text@v0.13.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	golang.org/x/net@v0.0.0-20211209124913-491a49abca63: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
	golang.org/x/text@v0.3.7: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
```

**Review by 0xFelix — APPROVED** [review]
/approve
/lgtm

**Pull request merged**